beebop/app.py

```diff
@@ -247,8 +252,14 @@ def generate_microreact_url_internal(microreact_api_new_url,
     r = requests.post(microreact_api_new_url,
                       data=json.dumps(json_microreact),
                       headers=headers)
-    url = r.json()['url']
-    return jsonify(response_success({"cluster": cluster, "url": url}))
+    try:
```

Comment:
should you also be throwing here based on the status code you get from the response?